test/integration/userStatus.test.js

```diff
@@ -404,7 +404,52 @@ describe("UserStatus", function () {
           }
           expect(res).to.have.status(400);
           expect(res.body.error).to.equal(`Bad Request`);
-          expect(res.body.message).to.equal(`"currentStatus.message" is not allowed to be empty`);
+          expect(res.body.message).to.equal(
+            `The value for the 'message' field is mandatory when State is OOO for more than three days.`
+          );
```

Comment:
can we be consistent in approaches, like the one in line 384

Reply:
changed line 387 to maintain consistency.